gloo/docs/user_guides/setup_options/observability.md

```diff
@@ -4,6 +4,90 @@ weight: 3
 description: How to monitor and trace within your Gloo setup.
 ---
 
+
+Gloo can be configured to provide observability into your system. Depending on your needs, some aspects of observability must be configured during setup. Please see below for details.
+
+- [Tracing Setup](#tracing)
+- [Metrics Setup](#metrics)
+
+# Tracing
+
+Gloo makes it easy to implement tracing on your system through [Envoy's tracing capabilities.
+](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing.html)
+
+#### Usage
+
+*If you have not yet enabled tracing, please see the [configuration](#configuration) details below.*
+
+- Produce a trace by passing the header: `x-client-trace-id`
+  - This id provides a means of associating the spans produced during a trace. The value must be unique, a uuid4 is [recommended](https://www.envoyproxy.io/docs/envoy/v1.9.0/configuration/http_conn_man/headers#config-http-conn-man-headers-x-client-trace-id).
+- Optionally annotate your trace with the `x-envoy-decorator-operation` header.
+  - This will be emitted with the resulting trace and can be a means of identifying the origin of a given trace. Note that it will override any pre-specified route decorator. Additional details can be found [here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/router_filter#config-http-filters-router-x-envoy-decorator-operation).
+
+#### Configuration
+
+- There are two steps to make tracing available through Gloo:
+  1. Gloo specify a trace provider in the Helm values specification.
+  1. Enable tracing on the listener.
+  1. (Optional) Annotate routes with descriptors.
+
+##### 1. Specify a tracing provider in Helm values
+
+Several tracing providers are supported. You can choose any that is supported by Envoy and pass your specific configuration to Gloo during installation as a Helm value specification.
+
+For a list of supported tracing providers and the configuration that they expect, please see Envoy's documentation on [trace provider configuration](https://www.envoyproxy.io/docs/envoy/v1.9.0/api-v2/config/trace/v2/trace.proto#config-trace-v2-tracing-http).
+For demonstration purposes, we show how to specify the helm values for a *zipkin* trace provider below.
+
+{{< highlight yaml "hl_lines=3-8" >}}
+gatewayProxies:
+  gatewayProxy:
+    tracing: |
+      name: envoy.zipkin
+      typed_config:
+        "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
+        collector_cluster: zipkin
+        collector_endpoint: "/api/v1/spans"
+{{< /highlight >}}
+
```

Comment:
I would add a note that you can update the envoy configuration post-install by updating the gateway-proxy-envoy-config config map.